api/src/test/resources/io/strimzi/api/kafka/model/040-Crd-kafka-crdApi-v1.yaml

```diff
@@ -1589,6 +1596,13 @@ spec:
                                     type: string
                                   user:
                                     type: string
+                              seccompProfile:
```

Comment:
from where this `seccompProfile` comes? what is that?

Reply:
New versions of Fabric8 often adopt newer Kubernetes schemas which have new fields. And that can change the CRDs in places where we use the Kubernetes APIs directly - here in this case the security context.